packages/components/src/shortcuts/NavigationShortcuts.ts

```diff
@@ -0,0 +1,35 @@
+import ShortcutRegistry from './ShortcutRegistry';
+import { MODIFIER, KEY } from './Shortcut';
+
+const NAVIGATION_SHORTCUTS = {
+  CYCLE_TO_NEXT_STACK: ShortcutRegistry.createAndAdd({
+    id: 'NAVIGATION.CYCLE_TO_NEXT_STACK',
+    name: 'Cycle To Next Stack',
+    shortcut: [MODIFIER.CTRL, KEY.SINGLE_QUOTE],
+    macShortcut: [MODIFIER.CMD, KEY.SINGLE_QUOTE],
+    isEditable: true,
+  }),
+  CYCLE_TO_PREVIOUS_STACK: ShortcutRegistry.createAndAdd({
+    id: 'NAVIGATION.CYCLE_TO_PREVIOUS_STACK',
+    name: 'Cycle To Previous Stack',
+    shortcut: [MODIFIER.CTRL, KEY.SEMICOLON],
+    macShortcut: [MODIFIER.CMD, KEY.SEMICOLON],
+    isEditable: true,
+  }),
+  CYCLE_TO_NEXT_TAB: ShortcutRegistry.createAndAdd({
+    id: 'NAVIGATION.CYCLE_TO_NEXT_TAB',
+    name: 'Cycle To Next Tab',
+    shortcut: [MODIFIER.CTRL, MODIFIER.SHIFT, KEY.SINGLE_QUOTE],
+    macShortcut: [MODIFIER.CMD, MODIFIER.SHIFT, KEY.SINGLE_QUOTE],
+    isEditable: true,
+  }),
+  CYCLE_TO_PREVIOUS_TAB: ShortcutRegistry.createAndAdd({
+    id: 'NAVIGATION.CYCLE_TO_PREVIOUS_TAB',
+    name: 'Cycle To Previous TAB',
+    shortcut: [MODIFIER.CTRL, MODIFIER.SHIFT, KEY.SEMICOLON],
```

Comment:
Hmm these Shifted keyboard shortcuts don't work for me on Linux. It needs to reference the double quotes and the colon instead (what's actually typed with the Shift). Those default shortcuts work for you on mac? Wonder if that's a bug in our code or it's a difference in how the OS passes us key events.

```suggestion
    shortcut: [MODIFIER.CTRL, MODIFIER.SHIFT, KEY.DOUBLE_QUOTES],
    macShortcut: [MODIFIER.CMD, MODIFIER.SHIFT, KEY.SINGLE_QUOTE],
    isEditable: true,
  }),
  CYCLE_TO_PREVIOUS_TAB: ShortcutRegistry.createAndAdd({
    id: 'NAVIGATION.CYCLE_TO_PREVIOUS_TAB',
    name: 'Cycle To Previous TAB',
    shortcut: [MODIFIER.CTRL, MODIFIER.SHIFT, KEY.COLON],
```

We'll want to get this tested on Windows as well.